src/main/scala/com/scality/clueso/SparkUtils.scala

```diff
@@ -0,0 +1,129 @@
+package com.scality.clueso
+
+import java.io.{File, IOException}
+import java.net.URI
+
+import com.scality.clueso.query.{MetadataQuery, MetadataQueryExecutor}
+import com.typesafe.config.ConfigFactory
+import com.typesafe.scalalogging.LazyLogging
+import org.apache.hadoop.conf.{Configuration => HadoopConfig}
+import org.apache.hadoop.fs.{FileStatus, FileSystem, Path, PathFilter}
+import org.apache.spark.sql.functions.{col, lit}
+import org.apache.spark.sql.{Row, SparkSession}
+
+
+object SparkUtils extends LazyLogging {
```

Comment:
is it good practice to just always extend off of this if you want logging?

Reply:
It's lazy evaluated so, if you're not logging, it won't have impact in runtime. Lots of scala programmers use it. The resulting bytecode is the same as LazyLogging trait is only adding and lazily initializing a logger on the first time the class uses one.